pkg/component/networking/apiserverproxy/apiserver_proxy_test.go

```diff
@@ -462,6 +462,12 @@ static_resources:
           "@type": type.googleapis.com/envoy.extensions.filters.network.tcp_proxy.v3.TcpProxy
           stat_prefix: kube_apiserver
           cluster: kube_apiserver
+		  tunneling_config:
```

Comment:
This not to seem correct: https://github.com/stackitcloud/gardener/pull/97/commits/a57ce23b3b16917161593165f55f543fd9beb88f#diff-08f1f63ed2f60c43bf365ed0febad215ee59b84095021725dceb61d95c878469